backend/src/main/java/capstone/myfinancemanager/manager/controller/TransactionController.java

```diff
@@ -37,6 +48,15 @@ public ResponseEntity<Void> deletePlant(@PathVariable String id) {
         return new ResponseEntity<>(deleteSuccess ? HttpStatus.NO_CONTENT : HttpStatus.NOT_FOUND);
     }
 
+    @PutMapping("/update/{id}")
```

Comment:
Not a resource oriented url (not restful). Combination of path and http verb is sufficient, please use just `@PutMapping("{id}")`